deployment/articles/100-reference/07-liferay-code-upgrade-plan.markdown

```diff
@@ -0,0 +1,245 @@
+# Liferay Code Upgrade Plan 
```

Comment:
Should this be `Liferay Plugins Upgrade Plan`?

Reply:
Thoughts @jhinkey @jtydhr88

Reply:
we also have Liferay Workspace code support, I don't think **Plugin** will cover it